pandas/core/categorical.py

```diff
@@ -602,6 +602,35 @@ def _get_categories(self):
     categories = property(fget=_get_categories, fset=_set_categories,
                           doc=_categories_doc)
 
+    def _codes_for_groupby(self, sort):
+        """
+        Return a Categorical adjusted for groupby
+
```

Comment:
adjusted is not very clear ; can you make this a bit more clear (and the comments below)

pretend you have never seen this and see it from a new reader perspective